system-parachains/people/people-polkadot/src/bridge_common_config.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
```

Comment:
@antkve please, move/join all these stuff directly to the `bridge_to_bulletin_config.rs` we don't need `bridge_common_config.rs` as standalone file